node/src/command.rs

```diff
@@ -438,7 +443,9 @@ pub fn run() -> Result<()> {
 		},
 		Some(Subcommand::PurgeChain(cmd)) => {
 			let runner = cli.create_runner(cmd)?;
+			let chain_spec = &runner.config().chain_spec;
 
+			set_default_ss58_version(chain_spec);
```

Comment:
Why need to `set_default_ss58_version` when purge chain?

Reply:
No specific reason. Just keep it the same with others. :)